README.md

```diff
@@ -150,6 +151,7 @@ additional details on each available environment variable.
 | `ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION` | 10m | Time to wait to delete containers for a stopped task. If set to less than 1 minute, the value is ignored.  | 3h | 3h |
 | `ECS_CONTAINER_STOP_TIMEOUT` | 10m | Instance scoped configuration for time to wait for the container to exit normally before being forcibly killed. | 30s | 30s |
 | `ECS_CONTAINER_START_TIMEOUT` | 10m | Timeout before giving up on starting a container. | 3m | 8m |
+| `ECS_CONTAINER_CREATE_TIMEOUT` | 10m | Timeout before giving up on creating a container. | 4m | 4m |
```

Comment:
We should mention here what minimum timeout is. Also, it would be helpful if there was a paragraph somewhere explaining how to find best value for use cases

Reply:
@mssrivas Unfortunately this will depend on the use case of the customer, for most customers default will be a good value.